dug_8/dug_8_dasharo_tools_suite_status.md

```diff
@@ -0,0 +1,421 @@
+    Book a call
+    </a>
+- <a href="https://newsletter.3mdeb.com/subscription/PW6XnCeK6">
+      Sign up for the newsletter
+    </a>
+]
+
+???
+
+- Time for this slide: 25s
+- Idea/goal of this slide: Present yourself
+- What to say: "My name is Daniil. I am junior embedded system developer
+interested in system level programming and security. Currently I am a member of
+3mdeb Zarhus team, and responsible, among other, for Dasharo Tools Suite."
+- Notes:
+
+---
+
+# Agenda
+
+.left-column50[
+* What is DTS?
+* DTS statistics Q3-Q4 2024.
+  + Issues
+  + Contributions.
+  + Features.
+  + Hardware support.
+* DTS current architecture and tests.
+* DTS new architecture and tests.
+* Roadmap Q1-Q2 2025
+* Q&A
+]
+
+???
+
+- Time for this slide: 35s
+- Idea/goal of this slide: Introduce agenda
+- What to say: "Here is the agenda for this meeting. After a brief intro into
+DTS we will take a look on DTS latest contribution and features statistics. Then
+we will discuss the transition between old and new DTS architecture. After the
+architecture a new testsuites will be presented and described. Then we will
+finish with questions and answers, where you will be able to ask questions and
+share your ideas."
+- Notes:
+
+---
+
+# What is DTS?
+
+.center[<img src="/img/solid-and-secure-bridge.png" height="280px">]
+
+.center[<font size="16">Solid and Secure bridge</font>]
+
+???
+
+- Time for this slide: 50s
+- Idea/goal of this slide: Present DTS
+- What to say: "What is DTS? DTS states for Dasharo Tools Souite and is a tiny
+Linux Kernel built for 64-bit x86with goal to make Dasharo user experience
+easier. Actually, it could be thought of as a solid and secure bridge between a
+user and Dasharo. It has several Dasharo and Coreboot tools built in, as well
+as other, firmware related tools for enhanced debugging and programming. But
+most importantly, it has a layer of automation and GUI scripts that make DTS
+more user-friendly."
+- Notes:
+
+---
+
+# DTS statistics
+
+.center[<img src="/img/dts-issues-q3-q4-2024.png" height="380px">]
+
+.center[Two big releases: 2.0.0 and 2.1.0!]
+
+???
+
+- Time for this slide: 45s
+- Idea/goal of this slide: Present DTS issues statistics
+- What to say: "There have been a lot of changes in DTS since first half of 2024
+year. We had several minor releases with hot fixes and, most importantly we had
+two big releases 2.0.0 and 2.1.0. Unfortunately, our plan to win in issues fight
+met a counterattack, and now we are fighting back. Here I would like to thank
+all of users who add issues and participate in discussions, it is really
+important for developers. Great thanks to all of you!"
+- Notes:
+
+---
+
+# DTS changes
+
+.left-column50[
+Updates:
+* Linux Kernel to 6.6.21
+* Updates for meta layers
+
+New features:
+* pre-commit now used everywhere
+* Some new tools
+* Verbose debugging mode
+* New GUI
+* DTS extensions feature
+* UEFI Capsule Update support
```

Comment:
Diagrams as well as process of UEFI Capsule Update are pretty complicated, IMO I do not think it is the right place for this. This should be presented as a part of Dasharo Firmware presentation,  here I will only mention that it has been integrated, what are the difference, and why it will be preferred way to update Dasharo.

Reply:
@DaniilKl I agree, but the point is to show things from a UX perspective. DTS UX is very important for Dasharo's success and for showcasing the Zarhus Team's mastery. Sooner or later, I hope Zarhus will have its event, and it will be 80% showcasing demos and how things work. I know it is too late to improve, but maybe next time, at least consider that.